src/main/java/qna/domain/Question.java

```diff
@@ -85,8 +85,27 @@ public boolean isDeleted() {
 		return deleted;
 	}
 
-	public void setDeleted(boolean deleted) {
-		this.deleted = deleted;
+	public void delete() {
+		this.deleted = true;
+		this.answers.stream()
+			.forEach(answer -> answer.setDeleted(true));
```

Comment:
다음과 같이 해볼 수 있을 것 같아요! 
또한 Answers 객체를 만들어 보는건 어떨까요? 😄 
```suggestion
		this.answers.forEach(answer -> answer.setDeleted(true));
```

Reply:
조언 감사합니다! 항상 stream을 쓰는게 습관이 되어서 길게 작성하게 되었네요..ㅠ 수정하였습니다!
일급컬렉션에 대한 것은 생각을 하려고 노력하는데 막상 jpa 와 같이 쓰려하면 잘 생각이 안납니다..
덕분에 항상 놓치던 부분을 수정할 수 있게 되어서 감사드립니다 😀